Podfile

```diff
@@ -117,12 +116,15 @@ def gutenberg_dependencies(options)
         'React-RCTSettings',
         'React-RCTText',
         'React-RCTVibration',
+        'React-callinvoker',
         'React-cxxreact',
         'React-jsinspector',
         'React-jsi',
         'React-jsiexecutor',
+        'React-perflogger',
+        'React-runtimeexecutor',
```

Comment:
Curious, are these new React modules that are part of the new version?

Reply:
That's right. Normally they are included via the new [react_native_pods](https://github.com/facebook/react-native/blob/v0.64.0/scripts/react_native_pods.rb#L47-L48) script in Gutenberg demo app's Podfile [here](https://github.com/WordPress/gutenberg/pull/29118/files#diff-9a2250a2cc0b1edbfbdf9cd4edc78323b6bccd9cf02689be56a6b7b90d065d26R5). But WPiOS still needs us to define each one manually.

Reply:
Thanks for explaining 👍